builder/bundleset.go

```diff
@@ -355,6 +359,21 @@ func parseBundle(contents []byte) (*bundle, error) {
 				return nil, fmt.Errorf("Invalid bundle name %q in line %d", text, line)
 			}
 			optional = append(optional, text)
+		} else if strings.HasPrefix(text, "content(") {
+			if !strings.HasSuffix(text, ")") {
+				return nil, fmt.Errorf("Missing end parenthesis in line %d: %q", line, text)
+			}
+			text = text[8 : len(text)-1]
+
+			// Remove unnecessary trailing slashes
+			for len(text) > 1 && strings.HasSuffix(text, "/") {
+				text = strings.TrimSuffix(text, "/")
```

Comment:
@reaganlo I added this condition to drop trailing slashes when parsing the bundle.